example/server.js

```diff
@@ -17,11 +17,9 @@ server.register({
     host: process.env.PAGEDATA_HOST || `http://localhost:${port}`,
     key: process.env.PAGEDATA_KEY || 'key',
     status: 'draft',
-    enablePageCache: process.env.PAGEDATA_CACHE || false,
-    enableProjectPagesCache: process.env.PAGEDATA_CACHE || false,
-    enableParentPagesCache: process.env.PAGEDATA_CACHE || false,
-    cacheEndpoint: '/cache',
-    hookEndpoint: '/hook',
+    pageCache: process.env.PAGEDATA_CACHE || false,
```

Comment:
lets set each of these to
`{ expiresIn: 60 * 1000, staleIn: 5000, staleTimeout: 200, generateTimeout: 5000 }`